docs/docs/cli-commands.md

```diff
@@ -1903,6 +1904,38 @@ In order to use [Netlify Dev](https://www.netlify.com/products/dev/) you need to
 
 > Note: To detect the RedwoodJS framework, please use netlify-cli v3.34.0 or greater.
 
+### setup package
+
+This command takes a published npm package that you specify, performs some compatibility checks, and then executes it's bin script. This allows you to use third-party packages that can provide you with an easy-to-use setup command for the particular functionality they provide.
+
+This command behaves similarly to `yarn dlx` but will attempt to confirm compatibility between the package you are attempting to run and the current version of Redwood you are running. You can bypass this check by passing the `--force` flag if you feel you understand any potential compatibility issues.
+
+```
+yarn redwood setup package <npm-package>
+```
+
+| Arguments & Options | Description              |
+| :------------------ | :----------------------- |
+| `--force, -f`       | Overwrite existing files |
+
+**Usage**
+
+Run the made up `@bluewoodjs/rw-setup` package:
```

Comment:
Let's change the scope to `@redwoodjs` here so that we at least know what happens if someone doesn't really read these docs and just blindly copy-pastes this codeblock:

```suggestion
Run the made up `@redwoodjs/setup-example` package:
```